tests/acl/test_acl.py

```diff
@@ -612,14 +612,6 @@ def test_ingress_unmatched_blocked(self, setup, direction, ptfadapter, ip_versio
         pkt = self.tcp_packet(setup, direction, ptfadapter, ip_version)
         self._verify_acl_traffic(setup, direction, ptfadapter, pkt, True, ip_version)
 
-    def test_egress_unmatched_forwarded(self, setup, direction, ptfadapter, ip_version, stage):
```

Comment:
Why removed this case?

Reply:
I don't think this case should be removed. I'd like to keep it.